src/mixins/menubar.js

```diff
@@ -167,6 +167,44 @@ export default [
 			return command.toggleBlockquote()
 		},
 	},
+	{
+		label: t('text', 'Callouts'),
+		visible: false,
+		children: [
+			{
+				label: t('text', 'Info'),
+				class: 'icon-info',
+				isActive: ['calloutContainer', { type: 'info' }],
+				action: (command) => {
+					return command.toggleCalloutCustomContainer({ type: 'info' })
+				},
+			},
+			{
+				label: t('text', 'Success'),
+				class: 'icon-success',
+				isActive: ['calloutContainer', { type: 'success' }],
+				action: (command) => {
+					return command.toggleCalloutCustomContainer({ type: 'success' })
+				},
+			},
+			{
+				label: t('text', 'Warning'),
+				class: 'icon-warn',
+				isActive: ['calloutContainer', { type: 'warn' }],
+				action: (command) => {
+					return command.toggleCalloutCustomContainer({ type: 'warn' })
+				},
+			},
+			{
+				label: t('text', 'Error'),
```

Comment:
```suggestion
				label: t('text', 'Danger'),
```

Maybe danger would be a bit less technical as the user facing term?